EndreProfil.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace UserObject\EndreProfil;
+
+use PDO;
+use UserObject\User;
+
+class EndreProfil
+{
+    public User $user;
+    public object $database;
+    public string $out = "";
+    
+    public function __construct(User $user, PDO $database)
+    {
+        $this->user = $user;
+        $this->database = $database;
+        $this->execute();
+    }
+    
+    private function execute()
+    {
+        if ($write = canUseFunction(1, 1)) {

```

Comment:
This function will hinder the user from using the function if jailed or bunkered. Either set values to 0 and 0 unless the intention is to not allow changing a profile while either option is active, or simply do "readyEndreProfil" without the if clause.